docs/en/basic/Command-Line-Interface.md

```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
 ./bin/alluxio runTests
 ```
 
+### runUfsTests
+
+The `runUfsTests` aims to test the integration between Alluxio and the given ufs. The ufs tests
+contain all the Alluxio semantics with the ufs.
+
+`--help` will provide the detail guidance on how to run this command.
+`--path <ufs_path>` is required to provide the full ufs path to run ufs tests against.
```

Comment:
`--path <ufs_path>` (required) the full UFS path to run tests against.

Reply:
Changed, thanks